src/loaders/lottie/tvgLottieLoader.cpp

```diff
@@ -302,7 +302,22 @@ Paint* LottieLoader::paint()
 
 bool LottieLoader::override(const char* slot)
 {
-    if (!slot || !comp || comp->slots.count == 0) return false;
+    if (!comp || comp->slots.count == 0) return false;
+
+    //reverting whole slots
+    if (!slot) {
+        if (!this->overriden) return true;
+
+        for (auto s = comp->slots.begin(); s < comp->slots.end(); ++s) {
+            for (auto pair = (*s)->pairs.begin(); pair < (*s)->pairs.end(); ++pair) {
+                if ((*pair).prop) (*pair).obj->revert((*pair).prop);
```

Comment:
like the assign() with my comments, you can just revert the value. (it would not require the new revert functions)